README.md

```diff
@@ -0,0 +1,49 @@
+# fusion-react
+
+FusionJS entry point for React universal rendering
+
+---
+
+### Installation
+
+```sh
+yarn add fusion-react
+```
+
+---
+
+### Example
+
+```js
+// ./src/main.js
+import React from 'react';
+import App from 'fusion-react';
+
+const Hello = () => <div>Hello</div>
+
+export default function() {
+  return new App(<Hello />);
+}
+```
+
+---
+
+### API
```

Comment:
Docs for HOC?

Reply:
Honestly I think we should maybe not document it at all. I'm not convinced we want to expose this interface

Reply:
my 2 cents: document, and if we change it, do it formally via a changelog/version bump